quora-service/src/main/java/com/upgrad/quora/service/business/UserService.java

```diff
@@ -0,0 +1,47 @@
+package com.upgrad.quora.service.business;
+
+import com.upgrad.quora.service.common.ErrorMessage;
+import com.upgrad.quora.service.dao.UserDao;
+import com.upgrad.quora.service.entity.UserEntity;
+import com.upgrad.quora.service.exception.AuthorizationFailedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+public class UserService {
```

Comment:
UserService class not needed .. there is already UserBusinessService present use that class

Reply:
Merged UserService class into UserBusinessService class. I have commented the code in UserService class, you can delete tne UserService class .. made changed to 3 class -
1. UserService
2. UserBusiessService
3. AdminController